source/parser.hera

```diff
@@ -675,7 +675,7 @@ PipelineHeadItem
 PipelineTailItem
   Await
   Yield
-  Return
+  Return !"." -> $1
```

Comment:
Mabye want to be AccessStart so we can pick up `?.`, etc.

Reply:
Not technically possible currently, because we currently explicitly check only for `return.value`, but `!AccessStart` seems safer against the future, also depending on how #744 works out.